### PR TITLE
Docker image

### PR DESCRIPTION
**Pull request by andreaspeters**
Hi Johni,

I have created a easy to use docker image for your mumble-web and want share it with you.

Regards,
Andreas

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    npm run build 
+
+USER root
+
+RUN apk del gcc git make g++
+
+USER node
+
+EXPOSE 8080
+ENV MUMBLE_SERVER=mumble.aventer.biz:64738
```
Comment (Johni0702) [inline Dockerfile]: Is it standard practice in docker-land to hard-code example values and have the user of the dockerfile replace them manually or is there some way to force the user to provide this piece of configuration before being able to run it?

Comment (andreaspeters) [reply Dockerfile]: Yes it is standard practice to defined example values with that the container will work. If u want to use the container with your server, u just tell it docker engine during the start command:
 
```
docker run -e MUMBLE_SERVER=myownserver.com:12345 mumble-web
```
So u see, its not really "hardcoded" and you don't have to change the dockerfile. :-)

Comment (Johni0702) [reply Dockerfile]: Alright, thanks for the PR.

Comment (andreaspeters) [reply Dockerfile]: your welcome and thanks for the merge.

**Pull request merged**